assets/vue/main.js

```diff
@@ -0,0 +1,5 @@
+import MainLayout from "./components/MainLayout.vue";
```

Comment:
Importing the Vue package here will allow us to avoid using the CDN.

```suggestion
import Vue from "vue";
import MainLayout from "./components/MainLayout.vue";

```